bin/development-server.js

```diff
@@ -63,7 +65,9 @@ app.use(express.static("public"));
 
 // Routes
 app.get("/", function(req, res) {
-  res.sendFile(path.join(__dirname, "../index.html"));
+  const tplFile = fs.readFileSync(path.join(__dirname, "../index.html"), "utf8");
+  const localProduction = getValue("firefox.localProduction");
```

Comment:
This looks great, except we shouldn't need a separate flag. When I'm testing production, I mean the real production environment, so you can just do `Mustache.render(tplFile, { isDevelopment: isDevelopment() })` and only render the `<link>` tag when it's not in dev. Because when it's not in dev, the styles should be separated into a CSS file